merge glossaries (#389)
ietf-rats-wg/draft-ietf-rats-corim: draft-ietf-rats-corim.md

in particular, CBOR diagnostic notation is defined in {{Section 8 of -cbor}}
and {{Section G of -cddl}}. Terms and concepts are always referenced as proper nouns, i.e., with Capital Letters.

### Glossary {#sec-glossary}

This document uses the following terms:

{: vspace="0"}

actual state:
: Claims that describe a Target Environment instance at a given point in time.
Endorsed Values and Evidence typically are Claims about actual state.
An Attester can consist of multiple components, with each component representing a particular scope of appraisal.
See also ({{Section 2 of -rats-endorsements}}).

Appraisal Claims Set (ACS):

> : A structure that holds ECTs that have been appraised.

Reviewer: ```suggestion
: A structure that holds ECTs ( for ECTs see below) that have been appraised.
```

Author: If you look through the entries, this is not the only one.
There are constant references between the glossary entries, even 1:n.

The ACS contains Attester state that has been authorized by Verifier processing and Appraisal Policy.

Appraisal Policy:
: A description of the conditions that, if met, allow acceptance of Claims.
Typically, the entity asserting a Claim should have knowledge, expertise, or context that gives credibility to the assertion.
Appraisal Policy resolves which entities are credible and under what conditions.
See also "Appraisal Policy for Evidence" in {{-rats-arch}}.

Attestation Results Set (ARS):

> : A structure that holds results of appraisal and ECTs that are to be conveyed to a Relying Party.

Author: ```suggestion
: A structure that holds results of appraisal and Environment-Claim Tuples that are to be conveyed to a Relying Party.
```

Environment-Claim Tuple (ECT):
: A structure containing a set of values that describe a Target Environment plus a set of Measurement / Claim values that describe properties of the Target Environment.
The ECT also contains Authority which identifies the entity that authored the ECT.

Instance ID:
: An identifier of an Environment that is unique to that Environment instance, such as the serial number of a hardware module.
See also {{Section 4.2.1 of -eat}}.

Measurement:
: A value associated with specific security characteristics of an Attester that influences the trustworthiness of that Attester.
The object of a Measurement could be the invariant part of a firmware component loaded into memory during startup, a run-time integrity check (RTIC), a file system object, or a CPU register.
A measured object is part of the Attester's Target Environment.
Expected, or "golden," Measurements are compiled as Reference Values, which are used by the Verifier to assess the trust state of the Attester.
See also {{TNC.Arch}}, and Section 9.5.5 of {{TPM2.Part1}}.

Reference Values:
: A set of values that represent the desired or undesired state of an Attester.

> Reference Values are compared against Evidence to determine the trustworthiness of the Attester.

Reviewer 2: ```suggestion
Reference Values are compared against Evidence to determine whether Attester state is corroborated by a Reference Value Provider.
```

The term "Attesting Environment" refers to the entity that collects and cryptographically signs such security metrics.
See also {{Section 3.1 of -rats-arch}}.

Environment-Claim Tuple (ECT):

> : A structure containing a set of values that describe a Target Environment plus a set of Measurement / Claim values that describe properties of the Target Environment.

Reviewer: Agree: When Attest Key Triple introduces keys, ECT may also contain set of keys.

> : A structure containing a set of values that describe a Target Environment plus a set of Measurement / Claim values that describe properties of the Target Environment.

Reviewer 2: Revised wording. Is this better?

> : A structure containing a set of values that describe a Target Environment plus a set of Measurement / Claim values that describe properties of the Target Environment.

Author: @nedmsmith, in your latest suggestion above is "target" a synonym of "Target Environment"?

> : A structure containing a set of values that describe a Target Environment plus a set of Measurement / Claim values that describe properties of the Target Environment.

Reviewer: I would prefer the original text from Thomas

> : A structure containing a set of values that describe a Target Environment plus a set of Measurement / Claim values that describe properties of the Target Environment.

Reviewer: I think, target environment is the right term to use here, there is nothing like solo `target` to debate about!

@nedmsmith do you agree, we can create a github issue otherwise?

> : A structure containing a set of values that describe a Target Environment plus a set of Measurement / Claim values that describe properties of the Target Environment.

Reviewer 2: Yes. I just don't think the wording is as good as it could be. But what's there isn't wrong.

The ECT also contains Authority which identifies the entity that authored the ECT.

Instance ID:
: An identifier of an Environment that is unique to that Environment instance, such as the serial number of a hardware module.
See also {{Section 4.2.1 of -eat}}.

Measurement:
: A value associated with specific security characteristics of an Attester that influences the trustworthiness of that Attester.
The object of a Measurement could be the invariant part of a firmware component loaded into memory during startup, a run-time integrity check (RTIC), a file system object, or a CPU register.
A measured object is part of the Attester's Target Environment.
Expected, or "golden," Measurements are compiled as Reference Values, which are used by the Verifier to assess the trust state of the Attester.
See also {{TNC.Arch}}, and Section 9.5.5 of {{TPM2.Part1}}.

reference state:

> : Claims that describe various alternative states of a Target Environment.

Reviewer: ```suggestion
: A set of values that represent the desired or undesired state of an Attester.
```

Author: this is gone with the wind now

> : Claims that describe various alternative states of a Target Environment.

Author: gone

> Reference Values Claims typically describe various possible states due to versioning, manufactruing practices, or supplier configuration options.

Reviewer: ```suggestion
Reference Values Claims typically describe various possible states due to versioning, manufacturing practices, or supplier configuration options.
```

> Reference Values Claims typically describe various possible states due to versioning, manufacturing practices, or supplier configuration options.

Author: gone

See also {{Section 2 of -rats-endorsements}}.

Reference Values:

> : A set of values that represent the desired or undesired state of an Attester.

Reviewer 2: Would it be better to just point to the definition in the endorsement draft?

> : A set of values that represent the desired or undesired state of an Attester.

Author: This is in line with RATS endorsements.

This document assumes that Verifier implementations may differ.
To facilitate the description of normative Verifier behavior, this document uses an abstract representation of Verifier internals.

The terms
Claim,
Environment-Claim Tuple (ECT),

> reference state,
> actual state,

Reviewer: ```suggestion

```

Author: good catch!
